src/Controls/tests/DeviceTests/Memory/MemoryTests.cs

```diff
@@ -38,6 +39,7 @@ void SetupBuilder()
 	[InlineData(typeof(Label))]
 	[InlineData(typeof(RefreshView))]
 	[InlineData(typeof(ScrollView))]
+	[InlineData(typeof(TimePicker))]
 	public async Task HandlerDoesNotLeak(Type type)
```

Comment:
I reworked this PR to use the new test pattern in 0e27ca4c70b1877627cbd2734dc49ccaf413dd05.

The test was failing without these changes, so it makes these PRs easier.